Small additions/edits to ruler and alertmanager docs (#2983)
grafana/mimir: docs/sources/operators-guide/architecture/components/alertmanager.md

### Tenant configurations

Each tenant has an Alertmanager configuration that defines notifications receivers and alerting routes.

> The Mimir Alertmanager uses the same [configuration file used by the Prometheus Alertmanager](https://prometheus.io/docs/alerting/latest/configuration/#configuration-file).

Reviewer: ```suggestion
The Mimir Alertmanager uses the same [configuration file](https://prometheus.io/docs/alerting/latest/configuration/#configuration-file) that the Prometheus Alertmanager uses.
```
